lib/ci-stack.ts

```diff
@@ -28,6 +28,13 @@ import { JenkinsSecurityGroups } from './security/ci-security-groups';
 import { JenkinsWAF } from './security/waf';
 import { FineGrainedAccessSpecs } from './compute/auth-config';
 
+enum DeploymentType {
+  BTR='BTR',
```

Comment:
Just to clarify for external community.
```suggestion
  BTR='BTR', // Build Test Release
```